docs/tools/new_tools_temp.md

```diff
@@ -0,0 +1,186 @@
+
+<span class="tips">**Tip:** We recommend using a virtual environment to avoid any Python dependency conflicts. We recommend the [Pipenv](https://github.com/pypa/pipenv/blob/master/README.md) tool to manage your virtual development environment.
+
+## Prerequisite
+
+- Python 3. Install for [Windows](https://docs.python.org/3/using/windows.html), [Linux](https://docs.python.org/3/using/unix.html) or [macOS](https://docs.python.org/3/using/mac.html).
+- Pip (if not included in your Python installation). [Install for all operating systems](https://pip.pypa.io/en/stable/installing/).
+- CMake. [Install version 3.18.1 or newer for all operating systems](https://cmake.org/install/).
+- Ninja [Install version 1.0 or newer for all operating systems](https://github.com/ninja-build/ninja/wiki/Pre-built-Ninja-packages).
+- One of the support toolchains listed [in the build tools overview](../build-tools/index.html).
+
+## Install
+
+Use pip to install:
+
+- To install the latest release:
+
+    ```
+    python -m pip install mbed-tools
+    ```
+
+- To install a specific version:
+
+    ```
+    python -m pip install mbed-tools==<version number in major.minor.patch format>
+    ```
+
+- To install a pre-release or development version:
+
+    ```
+    python -m pip install mbed-tools --pre
+    ```
+
+## Set environment variables
+
+<!--like what? when? why? should this be with installation instructions?-->
+
+To override default environment variables:
+
+```
+mbed-tools env <VARIABLE>=<value>
+```
+
+You can also use a `.env` file containing definitions in the same `<VARIABLE>=<value>` format. Note that environment variables set using the command line will override those listed in the file.
+
+## Upgrade
+
+Use pip to upgrade your version:
+
+```
+python -m pip install mbed-tools --upgrade
+```
+
+# Use
+
+Currently, the new Mbed Tools supports three boards: `K64F`, `DISCO_L475VG_IOT01A` and `NRF52840_DK`.
+
+For help:
+
+- To get help for all commands, use:
+
+    ```
+    mbed-tools --help
+    ```
+
+- To get help for a specific command, use `mbed-tools <command> --help`. For example, for helping with listing connected devices (the `devices` command):
+
+    ```
+    mbed-tools devices --help
+    ```
+
+## Create a project
+
+You can create a new project or create a local copy of one of our example applications.
+
+### Create a new project
+
+To create a new Mbed OS project in a specified path:
+
+- To create the project and download a new copy of Mbed OS (latest official release):
+
+    ```
+    mbed-tools init <PATH>
+    ```
+
+- To create a project without downloading a copy of Mbed OS (reuse an existing copy):
+
+    ```
+    mbed-tools init -c <PATH>
+    ```
+
+<!--this seems to be an absolute path - had to start with `~/`. Can they work with relative paths? can they work with "here" to tell mbed-tools to work in the current directory?-->
```

Comment:
`mbed-tools init -c .` can be used and also with relative paths like `mbed-tools init -c .\mbed-os-example-blinky`